pkg/cmd/codespace/rebuild.go

```diff
@@ -10,22 +10,24 @@ import (
 
 func newRebuildCmd(app *App) *cobra.Command {
 	var codespace string
+	var fullRebuild bool
 
 	rebuildCmd := &cobra.Command{
 		Use:   "rebuild",
 		Short: "Rebuild a codespace",
 		Args:  cobra.NoArgs,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return app.Rebuild(cmd.Context(), codespace)
+			return app.Rebuild(cmd.Context(), codespace, fullRebuild)
 		},
 	}
 
 	rebuildCmd.Flags().StringVarP(&codespace, "codespace", "c", "", "Name of the codespace")
+	rebuildCmd.Flags().BoolVar(&fullRebuild, "full", false, "Perform a full rebuild of the codespace")
```

Comment:
@cmuto09 @jkeech Do you have any thoughts on the name of this flag and its description?

Reply:
Would be nice to add a bit more description to what exactly a `full` rebuild means. Maybe something like:

> Perform a full rebuild a codespace by removing and re-pulling all docker images on your virtual machine

Flag-wise I think `--full` makes sense since that's what we call it elsewhere

Reply:
The added detail is helpful! This description appears when users run `gh cs rebuild --help`. Here's what it would look like with your proposal, it might be a tad long:

<img width="820" alt="gh cs rebuild --help" src="https://user-images.githubusercontent.com/19893438/199346007-2de27f87-654e-41f9-8cfc-eb4b1b5d1999.png">

Here's a shorter option and its preview:

> Perform a full rebuild of the codespace by clearing cached Docker images

<img width="941" alt="gh cs rebuild --help" src="https://user-images.githubusercontent.com/19893438/199347125-27951402-70bf-4aa7-b38f-48e877b5e37e.png">

Reply:
Great idea! Check this out:

```
Rebuilding recreates your codespace. Your code and any current changes will be
preserved. Your codespace will be rebuilt using your working directory's
dev container. A full rebuild also removes cached Docker images.

USAGE
  gh codespace rebuild [flags]

FLAGS
  -c, --codespace string   name of the codespace
      --full               perform a full rebuild
```

<img width="624" alt="gh cs rebuild --help" src="https://user-images.githubusercontent.com/19893438/199353252-636139e9-f637-4dcd-a6c4-7c361d022b11.png">

Reply:
Love it! Great call @cmbrose